SPECS-EXTENDED/kernel-rt/kernel-rt.spec

```diff
@@ -121,11 +121,12 @@ Requires:       audit
 %description tools
 This package contains the 'perf' performance analysis tools for Linux kernel.
 
-%package -n     python3-perf
+%package        python3-perf
 Summary:        Python 3 extension for perf tools
 Requires:       python3
+Conflicts:      python3-perf
```

Comment:
I'm not sure if it's so straightforward, since all of these subpackages conflict with each other. As in: we might need to list the subpackages from all kernel specs we have. And then keep it up-to-date with every new kernel spec.

Also, did we figure out, if we actually cannot these subpackages only from `kernel.spec`? Now I don't remember. That would simplify this mess a lot. I know we already don't produce these subpackages from some kernel specs anyway (example: `kernel-mshv.spec`).

Reply:
+1 on the additional overhead of maintaining the `Conflicts:`.
Also, should we add an additional `Conflicts` in the main kernel spec ?